setup.py

```diff
@@ -0,0 +1,25 @@
+import pathlib
+from setuptools import setup
+
+HERE = pathlib.Path(__file__).parent
+README = (HERE / "README.md").read_text()
+
+setup(
```

Comment:
If we find it agreeable, I recommend using [poetry](https://python-poetry.org/).